src/it/resources/com/google/checkstyle/test/chapter4formatting/rule4821onevariableperline/InputOneVariablePerDeclaration.java

```diff
@@ -79,7 +94,8 @@ void method1() {
               // violation below 'Only one variable definition per line allowed.'
               java.lang
                   .String string1; java.lang.String[]
-                          strings1;
+                      strings1;
+              // violation 2 lines above '.* incorrect indentation .*, expected .*: 10, 12, 14.'
```

Comment:
ok.
unrelated indenation is fixed, 
indentation violation that is on same line as `'Only one variable definition per line allowed` stays